apps/desktop/src-tauri/src/backend.rs

```diff
@@ -37,11 +37,19 @@ where
             .path_resolver()
             .resolve_resource("model")
             .expect("bad bundle");
-        bundled.dylib_dir = Some(
+
+        bundled.dylib_dir = Some(if cfg!(target_os = "macos") {
             app.path_resolver()
                 .resolve_resource("dylibs")
-                .expect("missing `apps/desktop/src-tauri/dylibs`"),
-        );
+                .expect("missing `apps/desktop/src-tauri/dylibs`")
+                .parent()
+                .expect("invalid path")
+                .join("frameworks")
```

Comment:
Won't this resolve e.g.

```
bloop.app/Contents/Resources/dylibs/
```

to this? :

```
bloop.app/Contents/Resources/frameworks
```

I would expect the `Frameworks` directory to be one level up, and with a capital `F`.